Pak-Je-Kans.md

```diff
@@ -0,0 +1,47 @@
+# Wat wordt er gemaakt
+---
+We willen een plek maken op Mijn Amsterdam genaamd Pak Je Kans voorzieningencheck, hier kunnen mensen zien waar ze recht op hebben. Deze armoedevoorzieningen worden zo persoonlijk mogelijk gefilterd. Dit doen we met de gegevens die we van de burger hebben en krijgen, als de burger toegeeft dat we zijn of haar egevens gebruiken om deze voorzieningen te laten zien. Het doel van deze omgeving is ervoor zorgen dat de burger beter zicht heeft op wat ze allemaal recht hebben. De burgers kunnen het ook direct aanvragen.
+
+# Wat moet er gemaakt worden in de Back-End
+---
+* Er moeten files worden gemaakt met voorzieningen en voorwaarden erin.
+* Er moeten functies komen die deze voorwaarden toepassen op de voorzieningen en deze doorgeven aan de front end waar nodig is.
+* Ook moeten er tests worden geschreven om te checken of het allemaal werkt.
+
+# Benodigdheden
+---
+
+### informatie
+
+* Voorzieningen : Wigger
+    1. Welke voorzieningen zijn er
+    2. Wat zijn de voorwaarden van de voorzieningen
+* design : Bram
+    1. hoe gaat het er uit zien 
+    2. wat moet waar komen
```

Comment:
Meestal zijn er nog wat feedbackrondes, tekstuele redactie en uitwerking in Invision